File: backend/PayProModulith/src/main/java/com/example/paypromodulith/userManager/domain/model/FileDto.java
package com.example.paypromodulith.userManager.domain.model;

import java.io.Serializable;
import java.time.Instant;
import java.time.LocalDate;
import java.util.UUID;
import lombok.*;

/** DTO for {@link com.example.paypromodulith.entity.File} */
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class FileDto implements Serializable {
  private UUID id;
  private Instant createdAt;
  private String name;
  private LocalDate dateFile;
  private UserDto user;
  private OrganisationDto organisation;
  private String link;
}

File: frontend/web/PayProWeb/src/app/model/FileDto.ts
import {UserDto} from './UserDto';
import {OrganisationDto} from './OrganisationDto';

export interface FileDto {
	id: string;
	createdAt: string;
	name: string;
	dateFile: string;
	user: UserDto;
	organisation: OrganisationDto;
	link: string;
}
